src/components/cards/CardBgWithImage.tsx

```diff
@@ -1,67 +0,0 @@
-import React, { ReactNode } from "react";
-
-interface CardBgWithImageProps {
-  cardStyle: "planning" | "combat";
-  lockedOrUnlockedUnits?: "locked" | "unlocked";
-  saturation: "oversaturated" | "normal" | "half" | "quarter" | "zero";
-  bgImage?: string;
-  children?: ReactNode;
-}
-
-export default function CardBgWithImage({
-  cardStyle,
-  lockedOrUnlockedUnits,
-  saturation,
-  bgImage,
-  children,
-}: CardBgWithImageProps) {
-  const bg = bgImage;
-
-  const blurLockedUnitInfo =
-    lockedOrUnlockedUnits === "locked" ? `blur-sm` : ``;
-
-  const dimensions =
-    cardStyle === "combat"
-      ? `w-14 sm:w-24 md:w-28 lg:w-44 xl:w-56 h-full`
-      : `h-28 w-32 sm:h-52 sm:w-52`;
-
-  let saturate;
-  switch (saturation) {
-    case "oversaturated":
-      saturate = "saturate-125";
-      break;
-    case "normal":
-      saturate = "saturate-100";
-      break;
-    case "half":
-      saturate = "saturate-50";
-      break;
-    case "quarter":
-      saturate = "saturate-25";
-      break;
-    case "zero":
-      saturate = "saturate-0";
-      break;
-    default:
-      saturate = "saturate-100";
-  }
```

Comment:
You could safe yourself one branch and do 

```
case "normal":
default:
  saturate: "saturate-100"
  break;
```

Switches allow you to _fallthrough_ branches. If you don't explicitly put `break` at the end of a branch, the code will continue to evaulate the rest of the branches and execute them if needed.

Also always use `break`